README.md

```diff
@@ -150,6 +159,27 @@ simp_bolt::config::modulepath: /path/to/modules
 To apply an existing manifest, `su` to the bolt user and execute
 `bolt apply <manifest> --nodes <NODE NAME(S)> --password --sudo-password`.
 
+### Within the SIMP Omni-Environment
```

Comment:
We probably don't want users to do this. They should be able to run `bolt` without modifying a SIMP server at all.

Also, they shouldn't be running this as `root` so I'm not sure how they would have access to the various files.

Reply:
`simp config` and `simpenv` (which I believe is the best/correct way to set up a SIMP environment) do not currently allow for the creation of environments in alternate locations that I am aware of.  I figured this would be the best solution for now.  I am going to put in a feature request for simp-utils requesting flexibility on location and ownership of environments.

Reply:
@m-morrone The files are just copied into place. The users should be encouraged to simply copy the files from `/usr/share/simp` until we can get a command in place to make it easy.

Reply:
/usr/share/simp/environment-skeleton has permissions 750, unlike most everything else in /usr/share/simp.  can this be changed to 755 or is there something in there that requires protecting?

Reply:
@m-morrone Yeah, no reason that has to be protected. If you could PR that into the repo, it would be appreciated.

Reply:
Submitted PR for simp-environment-skeleton permissions.